packages/git-semver-tags/test.js

```diff
@@ -160,3 +160,16 @@ it('should not allow package filter without lernaTags=true', function(done) {
     done();
   }, {package: 'bar-project'});
 });
+
+it('should work with tag prefix option', function(done) {
+  writeFileSync('test6', '');
+  shell.exec('git add --all && git commit -m"eigth commit"');
+  shell.exec('git tag ms/6.0.0');
+  shell.exec('git add --all && git commit -m"tenth commit"');
+  shell.exec('git tag ms/7.0.0');
```

Comment:
Can you also please add a tag where `ms/` is somewhere else in the tag.

The purpose is to ensure the code above is properly checking for the prefix at the beginning of the tag string.